ScattReportTest.java

```diff
@@ -0,0 +1,68 @@
+import org.junit.Test;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.After;
+
+/**
+ * ScattTest.java
+ *
+ * @author Kara Beason
+ * @author Cydney Caldwell
+ * @author Michelle Melton
+ */
+
+/**
+ * Class to test Scatt class.
+ *
+ * @author Kara Beason
+ * @author Cydney Caldwell
+ * @author Michelle Melton
+ * @version Feb 2017
+ */
+public class ScattReportTest
+{
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+
+    /**
+     * Set up before tests.
+     */
+    @Before
+    public void setUpStreams()
```

Comment:
For consistency, you might call this setUp(). It also looks like this is not necessary if you don't use class fields.